packages/ui/src/utils/formatters.ts

```diff
@@ -0,0 +1,11 @@
+import BN from 'bn.js'
+import { formatBalance } from '@polkadot/util'
+
+const decimals = 12
```

Comment:
```suggestion
const DECIMALS = 12
```
(aka convention to have constants all caps)